src/Extension/FluentExtension.php

```diff
@@ -85,6 +86,22 @@ class FluentExtension extends DataExtension
      */
     const INHERITANCE_MODE_ANY = 'any';
 
+    /**
+     * Fluent inheritance mode for CMS context
+     *
+     * @var string
+     * @config
+     */
+    private static string $cms_localisation_required = FluentExtension::INHERITANCE_MODE_ANY;
+
+    /**
+     * Fluent inheritance mode for frontend context
+     *
+     * @var string
+     * @config
+     */
+    private static string $frontend_publish_required = FluentExtension::INHERITANCE_MODE_FALLBACK;
```

Comment:
```suggestion
     * Fluent inheritance mode for CMS context
     */
    private static string $cms_localisation_required = FluentExtension::INHERITANCE_MODE_ANY;

    /**
     * Fluent inheritance mode for frontend context
     */
    private static string $frontend_publish_required = FluentExtension::INHERITANCE_MODE_FALLBACK;
```
`@config` doesn't do anything, and we don't need typehints in the PHPDoc when we're using strong typing on the property itself.